### PR TITLE
Update LICENSE and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dseevr**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ and this will be downloaded on `make start` to be injected into the VMs by ansib
 
 Unfortunately, as of this writing, due to security concerns we do not publish
```
Comment (erikh) [inline CONTRIBUTING.md]: this whole paragraph can be removed, actually. we publish at github.com/contiv/build now.

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2015 Cisco Systems Inc. All rights reserved.
+Copyright 2016 Cisco Systems Inc. All rights reserved.
```
Comment (erikh) [inline LICENSE]: sorry to be such a pain, but I think it should read:

`Copyright 2015,2016 Cisco Systems Inc. All rights reserved.`

**Comment by erikh** [comment]
Is this ready?

**Comment by dseevr** [comment]
Yep, my changes are all in place.  If nothing else needs updating, fire away.

**Comment by erikh** [comment]
THanks for your contribution!

**Pull request merged**